SPECS/uefistored.spec

```diff
@@ -42,9 +42,14 @@ make
 # symlink binary to varstored to let XAPI find it
 ln -s uefistored %{buildroot}%{_sbindir}/varstored
 
+# Install PK.auth
+mkdir -p %{buildroot}%{_prefix}/share/varstored/
+cp %{_sourcedir}/PK.auth %{buildroot}%{_prefix}/share/varstored/PK.auth
+
 %files
 %{_sbindir}/uefistored
 %{_sbindir}/varstored
+%{_prefix}/share/varstored/PK.auth
```

Comment:
`%{_prefix}/share` is best written `%{_datadir}` (stands for line 47 too)

You also need to add this before line 52:
```
%dir %{_datadir}/varstored
```

Else the `/usr/share/varstored` directory will not be owned by this RPM and will be left behind when you uninstall it or upgrade it to another version of the RPM where that directory is not used anymore.

BTW does it need to be `/usr/share/varstored`? Could it be `/usr/share/uefistored`?